java/src/jmri/jmrix/AbstractMRNodeTrafficController.java

```diff
@@ -103,9 +105,11 @@ public synchronized AbstractNode getNode(int index) {
     }
 
     /**
-     * Public method to identify a SerialNode from its node address Note: 'addr'
-     * is the node address, numbered from 0. Returns 'null' if a SerialNode with
-     * the specified address was not found
+     * Public method to identify a SerialNode from its node address.
+     * <p>
+     * Note: 'addr' is the node address, numbered from 0.
```

Comment:
Instead of a note, this needs to be an `@param` tag (Javadoc will warn about the missing param here).

Reply:
Done